cluster/gce/config-default.sh

```diff
@@ -287,12 +287,7 @@ export ENABLE_DNS_HORIZONTAL_AUTOSCALER="${KUBE_ENABLE_DNS_HORIZONTAL_AUTOSCALER
 #   none           - Not run node problem detector.
 #   daemonset      - Run node problem detector as daemonset.
 #   standalone     - Run node problem detector as standalone system daemon.
-if [[ "${NODE_OS_DISTRIBUTION}" == "gci" ]]; then
```

Comment:
so we are loosing the test coverage for the standalone mode? I think the hidden logic of defauling it to standalone on COS is wrong. But I worry that we are loosing test coverage.

Reply:
We aren't because standalone tests are currently launched using the node e2e runner, which doesn't use the cluster/* scripts.

Reply:
Standalone tests

https://testgrid.k8s.io/sig-node-release-blocking#node-kubelet-containerd-standalone-mode

They are all node e2e tests.

Reply:
For NPD, I think we lost test coverage in standalone mode due to this PR. In NPD standalone mode, we currently pull the tar files from `gs://kubernetes-release/node-problem-detector/`. See https://github.com/kubernetes/kubernetes/blob/d3d06c3c7e07c7c79ff46c0fc3b9f081ce6b0226/cluster/gce/gci/configure.sh#L299C99-L299C117.

But running `gsutil ls gs://kubernetes-release/node-problem-detector/`, there is no NPD `v0.8.13`, which is bumped by this PR. It only has NPD versions up to `v0.8.10`. But none of the release blocking tests failed.